src/product/views.py

```
@@ -0,0 +1,54 @@
+from django.db import transaction
+from rest_framework.views import APIView
+from rest_framework.permissions import AllowAny
```

Ruff (0.8.2)

`rest_framework.permissions.AllowAny` imported but unused

Remove unused import: `rest_framework.permissions.AllowAny`

(F401)